models/dodo/arbitrum/dodo_aggregator_arbitrum_trades.sql

```diff
@@ -1,5 +1,6 @@
 {{ config
 (
+    tags=['dunesql'], 
     alias = alias('aggregator_trades'),
     partition_by = ['block_date'],
```

Comment:
we're looking to increase partition sizes universally. can you introduce a `block_month` column in the model (right next to `block_date`, same logic, but month level) and make it the partition value?